Walkthroughs implementation (#1698)
Microsoft/vscode-react-native: resources/walkthougths/debug-tutor.md

# Run and Debug your React Native application

> ## Ways to start debug session

Reviewer: ```suggestion
## Create and run debugging scenario
```

This extension supports generating debugging configuration in the `.vscode/launch.json` file by selecting debugging parameters in dropdown lists at the top of the editor.

![Add React Native debug configuration](../images/add-debug-configuration.gif)

> 

Reviewer: ```suggestion

To start debugging, just choose the appropriate configuration in the Configuration dropdown, and then press F5 (or click _Green Arrow_ ![Configure-gear](resources/images/debug-icon.png) button) to start debugging your app in VS Code.

```

# Run and Debug your React Native application
## Create and run debugging scenario
This extension supports generating debugging configuration in the `.vscode/launch.json` file by selecting debugging parameters in dropdown lists at the top of the editor.

![Add React Native debug configuration](../images/add-debug-configuration.gif)

To start debugging, just choose the appropriate configuration in the Configuration dropdown, and then press F5 (or click _Green Arrow_ ![Configure-gear](../images/debug-icon.png) button) to start debugging your app in VS Code.

The extension also allows to start debugging without creating the `launch.json` file in one of the following ways:
- Using dynamic debugging configurations

    ![Run dynamic debugging configuration](../images/dynamic-debugging-configuration.gif)
- Using Debug button in the Editor Title area menu

    ![Select and run debugging command](../images/debug-commands-button.png)

> ## Enable `Debug JS Remotely` and attach to application

Reviewer: ```suggestion
## Enable debugging mode and attach to the application
```
